kingfisher_scrapy/spiders/italy_anac.py

```diff
@@ -1,7 +1,7 @@
 import scrapy
```

Comment:
json_dumps is only needed in some narrow cases.

```suggestion
import json

import scrapy
```